geometry_msgs/Pose[] waypoints
---
string code
Route[] routes
